certified-connectors/IA-Connect Session/apiDefinition.swagger.json

```diff
@@ -3927,68 +3939,591 @@
+      }
+    },
+    "/Environment/StopKeepSessionAlive": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Stop keep session alive",
+        "description": "Stops the IA-Connect Agent user session keep alive.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "StopKeepSessionAlive",
+            "description": "Inputs for IA-Connect StopKeepSessionAlive action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}
+          }
+        },
+        "operationId": "StopKeepSessionAlive"
+      }
+    },
+    "/Environment/CopyFileToClipboard": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "summary": "Copy file to clipboard",
+        "description": "Copies the specified local file to the clipboard in the IA-Connect Agent user session.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "CopyFileToClipboard",
+            "description": "Inputs for IA-Connect CopyFileToClipboard action",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Filepath": {
+                  "type": "string",
+                  "title": "File name",
+                  "description": "The file path to the local file to be copied to the clipboard."
+                },
+                "Cut": {
+                  "type": "boolean",
+                  "default": false,
+                  "title": "Cut",
+                  "description": "If set to true then the file will be cut instead of copied."
+                },
+                "Workflow": {
+                  "type": "string",
+                  "x-ms-visibility": "advanced",
+                  "default": "@{workflow()}",
+                  "title": "Workflow",
+                  "description": "Add the following expression here: workflow()"
+                }
+              },
+              "required": [
+                "Filepath",
+                "Workflow"
+              ]
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {}

```

Comment:
@LukeMcGovern-UL Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
Updated output schema as requested.